spec/requests/configuration_scripts_spec.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.describe 'Configuration Scripts API' do
+  describe 'GET /api/configuration_scripts' do
+    it 'lists all the configuration scripts with an appropriate role' do
+      script = FactoryGirl.create(:configuration_script)
+      api_basic_authorize collection_action_identifier(:configuration_scripts, :read, :get)
+
+      get(api_configuration_scripts_url)
+
+      expected = {
+        'count'     => 1,
+        'subcount'  => 1,
+        'name'      => 'configuration_scripts',
+        'resources' => [
+          hash_including('href' => api_configuration_script_url(nil, script))
+        ]
+      }
+      expect(response.parsed_body).to include(expected)
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'forbids access to configuration scripts without an appropriate role' do
```

Comment:
@mkanoor - So we are enforcing RBAC with the feature?

Reply:
Yes